### PR TITLE
Address #24

### PR DESCRIPTION
**Pull request by caldwellst**
Adds short info about potential errors in pytest running from the incorrect environment.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -43,6 +43,15 @@ To run the tests and view coverage, execute:
 
    pytest --cov=aatoolbox
```
Comment (turnerm) [inline CONTRIBUTING.rst:44]: I would just swap out this command with the one you specified below, you can then give a warning that "If you run it directly with pytest, you may get the following error..." or something

**Review by turnerm — APPROVED** [review]
Thanks so much for taking care of this so quickly! Just one small suggestion

File: CONTRIBUTING.rst (hunk 1 above)
Comment (caldwellst) [reply CONTRIBUTING.rst]: Good idea! Addressed!

**Comment by caldwellst** [comment]
Okay, see what you think of the new text!

**Comment by turnerm** [comment]
That's perfect, good to merge any time!

**Pull request merged**